src/main/groovy/org/gradle/android/workarounds/RoomSchemaLocationWorkaround.groovy

```diff
@@ -188,23 +252,32 @@ class RoomSchemaLocationWorkaround implements Workaround {
         }
     }
 
-    private static void applyToAllAndroidVariants(Project project, Closure<?> configureVariant) {
+    private static void applyToAllAndroidVariants(Project project, ConfigureVariants configureVariants) {
+        if (Versions.CURRENT_ANDROID_VERSION <= VersionNumber.parse("7.4.0-alpha01")) {
```

Comment:
is this new API only available in 7.4? any downsides to using it 7.3 if it is available there?

Reply:
Only what you were communicating the other day that the new api doesn't work properly on some earlier versions.  Not knowing what versions it works properly with or not, I figured we would just use it starting with the version where we know we have issues with the old api.